eskip/eskip.go

```diff
@@ -334,6 +336,11 @@ func Parse(code string) ([]*Route, error) {
 	return routeDefinitions, nil
 }
 
+// Parses a route expression or a routing document to a set of route definitions.
```

Comment:
According the https://golang.org/doc/effective_go.html#commentary comments should start with function name:

```suggestion
// ParseBytes parses a route expression or a routing document to a set of route definitions.
```

It some languages, to generate docs, function name is ignored, that's why doc comments starts with function name.